index.html

```diff
@@ -205,5 +205,31 @@ <h2 class="about-me-topper">About me</h2>
         </div>
 
     </section>
-    </body>
-    </html>
\ No newline at end of file
+
+    <section class="form">
+        <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+        <form class="contact_form" action="https://formspree.io/f/mayzodlp" method="post">
+            <ul class="form-inputs">
+                <li><input type="text" placeholder="Full name" required maxlength="30" id="name" name="user_name"></li>
+                <li><input type="email" placeholder="Email address" id="email" name="user_email"></li>
+                <li><textarea id="message" name="user_message" maxlength="30" placeholder="Enter text here"></textarea>
```

Comment:
- [ ] As mentioned above please add the required attribute on the textarea element too currently we can submit the form with an empty message
- [ ] Also please correct the value of maxlength attribute for the textarea or messages it is supposed to be 500